src/google/protobuf/stubs/internal/status.cc

```diff
@@ -37,61 +37,64 @@
 namespace google {
 namespace protobuf {
 namespace util {
-namespace error {
```

Comment:
What's the rationale for moving this to a new `internal` subdirectory?

Reply:
Just stylistic preference. We can keep the implementation in this file if you'd like, with or without namespace `status_internal` (although I'd prefer keeping the namespace to already give a hint that the symbols will come from a different namespace eventually)

Reply:
OK, I think it would be good to keep the code in the same files but moving them into a `status_internal` namespace sounds good.

Reply:
Alright, done!